src/screens/time/components/EditPunchInOutDateTimeCardComponent.tsx

```diff
@@ -0,0 +1,275 @@
+              <TouchableOpacity
+                style={[
+                  styles.justifyContentCenter,
+                  styles.rowFlexDirection,
+                  {
+                    marginTop: theme.spacing,
+                    marginBottom: theme.spacing * 4,
+                  },
+                ]}
+                onPress={() => {
+                  this.showDatepicker();
+                }}>
+                <View
+                  style={[
+                    styles.rowFlexDirection,
+                    styles.flexOne,
+                    {
+                      paddingLeft: theme.spacing * 3,
+                    },
+                  ]}>
+                  <View
+                    style={{
+                      padding: theme.spacing,
+                      paddingRight: theme.spacing * 3,
+                    }}>
+                    <Icon name={'calendar-blank'} fontSize={20} />
+                  </View>
+                  <View>
+                    <Text style={{fontSize: theme.spacing * 4.5}}>
+                      {'Date'}
+                    </Text>
+                  </View>
+                </View>
+                <View style={[styles.rowFlexDirection, styles.flexTwo]}>
+                  <View
+                    style={[
+                      styles.alignItemsFlexEnd,
+                      styles.flexOne,
+                      {
+                        paddingRight: theme.spacing,
+                      },
+                    ]}>
+                    <FormattedDate
+                      style={{
+                        color: theme.palette.secondary,
+                        fontSize: theme.spacing * 4.5,
+                      }}>
+                      {dateDisplay}
+                    </FormattedDate>
+                  </View>
+                  <View style={[styles.alignItemsFlexEnd]}>
+                    <Icon name={'chevron-right'} fontSize={theme.spacing * 5} />
+                  </View>
+                </View>
+              </TouchableOpacity>
+              <Divider />
+              <TouchableOpacity
+                style={[
+                  styles.rowFlexDirection,
+                  styles.justifyContentCenter,
+                  {
+                    marginTop: theme.spacing * 4,
+                    marginBottom: theme.spacing * 5,
+                  },
+                ]}
+                onPress={() => {
+                  this.showTimepicker();
+                }}>
+                <View
+                  style={[
+                    styles.rowFlexDirection,
+                    styles.flexOne,
+                    {
+                      paddingLeft: theme.spacing * 3,
+                    },
+                  ]}>
+                  <View
+                    style={{
+                      padding: theme.spacing,
+                      paddingRight: theme.spacing * 3,
+                    }}>
+                    <Icon name={'clock-outline'} fontSize={20} />
```

Comment:
use `theme.typography.[suitable icon font size]` for `fontSize`, or remove `fontSize` if this prop not working